src/main/java/com/github/krzysiekjodlowski/ox/Player.java

```diff
@@ -0,0 +1,61 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+import com.github.krzysiekjodlowski.ox.ui.UI;
+
+/**
+ * Base for any player. Each one descendant
+ * will have to implement its own version
+ * of making moves and adjust toString.
+ *
+ * @author Krzysztof Jodlowski
+ */
+abstract class Player {
+  private final Symbol playersSymbol;
```

Comment:
symbol instead of playersSymbol would be enough. We are in Players class after all, everything here is "player's"

Reply:
Good point! Changed.